junit-jupiter-engine/src/main/java/org/junit/jupiter/engine/execution/ParameterResolutionUtils.java

```diff
@@ -135,6 +135,7 @@ private static Object resolveParameter(ParameterContext parameterContext, Execut
 			ParameterResolver resolver = matchingResolvers.get(0);
 			Object value = resolver.resolveParameter(parameterContext, extensionContext);
 			validateResolvedType(parameterContext.getParameter(), value, executable, resolver);
+			recordResolvedParameter(extensionContext, value);
```

Comment:
This would also record parameters for methods invoked via `ExtensionContext.getExecutableInvoker()`, right? I think we need to ensure that we only record them for the test method that's being executed, e.g. by checking `executable` equals `extensionContext.getTestMethod()`.